charts/karpenter/README.md

```diff
@@ -0,0 +1,45 @@
+# karpenter
```

Comment:
Was this fully autogenerated (as suggested by PR title)? I'm not sure where the domain specific details come from.

I want to make sure that we have the proper automation in place to keep this file updated (e.g. 0.5.0 -> 0.5.1).

Thoughts?

Reply:
user needs to run helm-docs to generate it. the github action CI is just checking if the readme is staled and needed to be re-generated in CI.

Reply:
the version is from this bit `{{ template "chart.version" . }}` in the `README.md.gotmpl`  or this `{{ template "chart.versionBadge" . }}`

Reply:
Ahh cool!